data/kg/compound_chebi/meta.yaml

```diff
@@ -0,0 +1,51 @@
+---
+name: compound_chebi
+description: Knowledgegraph data samples.
+targets:
+    - id: node2_type
+      description: node2_type
+      type: Other
+      units: node2_type
+      names:
+          - noun: node2_type
+    - id: node2_name
+      description: node2_name
+      type: Other
+      units: node2_name
+      names:
+          - noun: node2_name
+    - id: node2_id
+      description: node2_id
+      type: Other
+      units: node2_id
+      names:
+          - noun: node2_id
+identifiers:
+    - id: node1_type
+      description: node1_type
+      type: Other
+    - id: node1_smiles
+      description: node1_smiles
+      type: Other
+    - id: node1_name
+      description: node1_name
+      type: Other
+    - id: node1_id
+      description: node1_id
+      type: Other
+    - id: rel1_type
+      description: rel1_type
+      type: Other
+license: CC BY 4.0
+links:
+    - url: https://crossbar.kansil.org
+      description: original knowledge graph web GUI link
+num_points: 0
```

Comment:
oh, this was actually a bug because I was reading only the header of the csv and not the entire csv (to be faster).
I switched it back to reading the full csv and now the number of points are correct.